django/contrib/admin/options.py

```diff
@@ -565,7 +565,7 @@ class ModelAdmin(BaseModelAdmin):
     save_on_top = False
     paginator = Paginator
     preserve_filters = True
-    inlines = []
+    inlines = ()
```

Comment:
This change is backward incompatible. On the other hand users, `inlines` should be overwritten, not modified, by subclasses as explicitly stated in docs. So theoretically it won't affect anyone. I'm torn. 

If we want to move it forward we should do the same with `actions`.